mvvm/src/main/java/com/thefuntasty/mvvm/ViewModelFragment.kt

```diff
@@ -0,0 +1,35 @@
+package com.thefuntasty.mvvm
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.CallSuper
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProviders
+import com.thefuntasty.mvvm.delegate.ViewModelFragmentDelegate
+
+abstract class ViewModelFragment<VM : BaseViewModel<VS>, VS : ViewState> : Fragment(),
+    ViewModelFragmentDelegate<VM, VS> {
+
+    override lateinit var viewModel: VM
+
+    @CallSuper
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        viewModel = initViewModel(this)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/4a02c207953ea3674887f7e1a945e99f6b2b8a76/mvvm/src/main/java/com/thefuntasty/mvvm/ViewModelFragment.kt#L23"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>